tests/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ if(UNIX AND NOT APPLE)
 	set_target_properties(EndlessSkyTests PROPERTIES OUTPUT_NAME "endless-sky-tests")
 endif()
 
+if(NOT MSVC)
+	target_compile_options(EndlessSkyTests PUBLIC
+		"-Wall" "-pedantic-errors" "-Wold-style-cast" "-fno-rtti")
+endif()
```

Comment:
what is this about and how is it related to this PR

Reply:
I added a new unit test for ScrollVar, which uses virtual methods. Previously, EndlessSkyLib was built with -fno-rtti, and the unit test executable was not, so the linker was producing a warning. 

I'm pretty sure that we want the unit tests and the main code linked witht he same flags, so I added this cmake blurb. 

See comments at https://github.com/endless-sky/endless-sky/pull/9645#discussion_r1439080487